<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>930</width>
                <height>682</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QComboBox" name="fileBox">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>50</y>
                        <width>191</width>
                        <height>32</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLabel" name="myImage">
                <property name="geometry">
                    <rect>
                        <x>20</x>
                        <y>90</y>
                        <width>281</width>
                        <height>221</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Obraz wygenerowany</string>
                </property>
                <property name="alignment">
                    <set>Qt::AlignCenter</set>
                </property>
            </widget>
            <widget class="QPushButton" name="generateButton">
                <property name="geometry">
                    <rect>
                        <x>400</x>
                        <y>20</y>
                        <width>131</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Generuj obraz</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_2">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>20</y>
                        <width>161</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Wybierz obraz</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_3">
                <property name="geometry">
                    <rect>
                        <x>230</x>
                        <y>380</y>
                        <width>121</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Predykcja</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_4">
                <property name="geometry">
                    <rect>
                        <x>180</x>
                        <y>410</y>
                        <width>71</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Poz +</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_5">
                <property name="geometry">
                    <rect>
                        <x>270</x>
                        <y>410</y>
                        <width>71</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Neg -</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_6">
                <property name="geometry">
                    <rect>
                        <x>20</x>
                        <y>470</y>
                        <width>91</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Klasyfikacja</string>
                </property>
            </widget>
            <widget class="QLabel" name="labelTP">
                <property name="geometry">
                    <rect>
                        <x>180</x>
                        <y>450</y>
                        <width>68</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>0</string>
                </property>
            </widget>
            <widget class="QLabel" name="labelFN">
                <property name="geometry">
                    <rect>
                        <x>270</x>
                        <y>450</y>
                        <width>68</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>0</string>
                </property>
            </widget>
            <widget class="QLabel" name="labelFP">
                <property name="geometry">
                    <rect>
                        <x>180</x>
                        <y>490</y>
                        <width>68</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>0</string>
                </property>
            </widget>
            <widget class="QLabel" name="labelTN">
                <property name="geometry">
                    <rect>
                        <x>270</x>
                        <y>490</y>
                        <width>68</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>0</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_11">
                <property name="geometry">
                    <rect>
                        <x>120</x>
                        <y>450</y>
                        <width>41</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Poz +</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_12">
                <property name="geometry">
                    <rect>
                        <x>120</x>
                        <y>490</y>
                        <width>51</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Neg -</string>
                </property>
            </widget>
            <widget class="QProgressBar" name="generateBar">
                <property name="geometry">
                    <rect>
                        <x>400</x>
                        <y>60</y>
                        <width>131</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="value">
                    <number>0</number>
                </property>
            </widget>
            <widget class="QLabel" name="expertImage">
                <property name="geometry">
                    <rect>
                        <x>320</x>
                        <y>90</y>
                        <width>281</width>
                        <height>221</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Obraz eksperta</string>
                </property>
                <property name="alignment">
                    <set>Qt::AlignCenter</set>
                </property>
            </widget>
            <widget class="QLabel" name="label">
                <property name="geometry">
                    <rect>
                        <x>350</x>
                        <y>380</y>
                        <width>111</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Czułość (Sen)</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_7">
                <property name="geometry">
                    <rect>
                        <x>350</x>
                        <y>420</y>
                        <width>131</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Swoistość (Spec)</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_8">
                <property name="geometry">
                    <rect>
                        <x>350</x>
                        <y>460</y>
                        <width>131</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Dokładność (Acc)</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_9">
                <property name="geometry">
                    <rect>
                        <x>350</x>
                        <y>500</y>
                        <width>121</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Precyzja (Prec)</string>
                </property>
            </widget>
            <widget class="QLabel" name="labelSen">
                <property name="geometry">
                    <rect>
                        <x>480</x>
                        <y>380</y>
                        <width>68</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>0</string>
                </property>
            </widget>
            <widget class="QLabel" name="labelSpec">
                <property name="geometry">
                    <rect>
                        <x>480</x>
                        <y>420</y>
                        <width>68</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>0</string>
                </property>
            </widget>
            <widget class="QLabel" name="labelAcc">
                <property name="geometry">
                    <rect>
                        <x>480</x>
                        <y>460</y>
                        <width>68</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>0</string>
                </property>
            </widget>
            <widget class="QLabel" name="labelPrec">
                <property name="geometry">
                    <rect>
                        <x>480</x>
                        <y>500</y>
                        <width>68</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>0</string>
                </property>
            </widget>
            <widget class="QPushButton" name="myImageButton">
                <property name="geometry">
                    <rect>
                        <x>110</x>
                        <y>320</y>
                        <width>99</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Wyświetl</string>
                </property>
            </widget>
            <widget class="QPushButton" name="expertImageButton">
                <property name="geometry">
                    <rect>
                        <x>410</x>
                        <y>320</y>
                        <width>99</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Wyświetl</string>
                </property>
            </widget>
            <widget class="QPushButton" name="kfoldButton">
                <property name="geometry">
                    <rect>
                        <x>730</x>
                        <y>230</y>
                        <width>151</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Skrośna walidacja</string>
                </property>
            </widget>
            <widget class="QProgressBar" name="learnBar">
                <property name="geometry">
                    <rect>
                        <x>720</x>
                        <y>270</y>
                        <width>171</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="value">
                    <number>0</number>
                </property>
            </widget>
            <widget class="QLabel" name="accuracyLabel">
                <property name="geometry">
                    <rect>
                        <x>720</x>
                        <y>580</y>
                        <width>68</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>0</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_10">
                <property name="geometry">
                    <rect>
                        <x>720</x>
                        <y>310</y>
                        <width>171</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Liczba próbek na obraz</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_13">
                <property name="geometry">
                    <rect>
                        <x>720</x>
                        <y>370</y>
                        <width>171</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Liczba obrazów</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_14">
                <property name="geometry">
                    <rect>
                        <x>720</x>
                        <y>430</y>
                        <width>161</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Liczba K</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_15">
                <property name="geometry">
                    <rect>
                        <x>720</x>
                        <y>490</y>
                        <width>161</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Wielkość podobrazu</string>
                </property>
            </widget>
            <widget class="QSpinBox" name="probCount">
                <property name="geometry">
                    <rect>
                        <x>720</x>
                        <y>330</y>
                        <width>71</width>
                        <height>32</height>
                    </rect>
                </property>
                <property name="maximum">
                    <number>10000</number>
                </property>
                <property name="value">
                    <number>100</number>
                </property>
            </widget>
            <widget class="QSpinBox" name="imageCount">
                <property name="geometry">
                    <rect>
                        <x>720</x>
                        <y>390</y>
                        <width>71</width>
                        <height>32</height>
                    </rect>
                </property>
                <property name="value">
                    <number>5</number>
                </property>
            </widget>
            <widget class="QSpinBox" name="kCount">
                <property name="geometry">
                    <rect>
                        <x>720</x>
                        <y>450</y>
                        <width>71</width>
                        <height>32</height>
                    </rect>
                </property>
                <property name="value">
                    <number>10</number>
                </property>
            </widget>
            <widget class="QSpinBox" name="boxSize">
                <property name="geometry">
                    <rect>
                        <x>720</x>
                        <y>510</y>
                        <width>71</width>
                        <height>32</height>
                    </rect>
                </property>
                <property name="value">
                    <number>49</number>
                </property>
            </widget>
            <widget class="QLabel" name="label_16">
                <property name="geometry">
                    <rect>
                        <x>720</x>
                        <y>550</y>
                        <width>171</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Średnia dokładność</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_17">
                <property name="geometry">
                    <rect>
                        <x>250</x>
                        <y>20</y>
                        <width>131</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Rozmiar Obrazu</string>
                </property>
            </widget>
            <widget class="QSpinBox" name="heightValue">
                <property name="geometry">
                    <rect>
                        <x>230</x>
                        <y>50</y>
                        <width>61</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="minimum">
                    <number>1</number>
                </property>
                <property name="maximum">
                    <number>10000</number>
                </property>
                <property name="value">
                    <number>1000</number>
                </property>
            </widget>
            <widget class="QSpinBox" name="widthValue">
                <property name="geometry">
                    <rect>
                        <x>320</x>
                        <y>50</y>
                        <width>61</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="minimum">
                    <number>1</number>
                </property>
                <property name="maximum">
                    <number>9999</number>
                </property>
                <property name="value">
                    <number>1000</number>
                </property>
            </widget>
            <widget class="QLabel" name="label_18">
                <property name="geometry">
                    <rect>
                        <x>210</x>
                        <y>60</y>
                        <width>21</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>H:</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_19">
                <property name="geometry">
                    <rect>
                        <x>300</x>
                        <y>60</y>
                        <width>21</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>W:</string>
                </property>
            </widget>
            <widget class="QCheckBox" name="balancedBox">
                <property name="geometry">
                    <rect>
                        <x>350</x>
                        <y>530</y>
                        <width>191</width>
                        <height>25</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Oblicz zrównoważoną</string>
                </property>
            </widget>
            <widget class="QPushButton" name="loadModelButton">
                <property name="geometry">
                    <rect>
                        <x>790</x>
                        <y>20</y>
                        <width>121</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Załaduj</string>
                </property>
            </widget>
            <widget class="QComboBox" name="modelList">
                <property name="geometry">
                    <rect>
                        <x>560</x>
                        <y>20</y>
                        <width>211</width>
                        <height>32</height>
                    </rect>
                </property>
            </widget>
            <widget class="QPushButton" name="predictButton">
                <property name="geometry">
                    <rect>
                        <x>790</x>
                        <y>70</y>
                        <width>121</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Wysegmentuj</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_20">
                <property name="geometry">
                    <rect>
                        <x>350</x>
                        <y>570</y>
                        <width>201</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Średnia dokładność (acc):</string>
                </property>
            </widget>
            <widget class="QLabel" name="avgAcc">
                <property name="geometry">
                    <rect>
                        <x>350</x>
                        <y>600</y>
                        <width>68</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>0</string>
                </property>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>930</width>
                    <height>26</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
